exporter.go

```diff
@@ -165,11 +166,17 @@ func main() {
 			"telemetry.path", "URL path for surfacing collected metrics.").Default("/metrics").String()
 
 		perfObjects = kingpin.Flag(
-			"perflib.objects", "List of perflib objects to queryBuf (defaults to a built-in list)").Uint32List()
+			"perflib.objects", "List of perflib object indices to queryBuf (defaults to a built-in list)").Uint32List()
 		perfObjectsAdd = kingpin.Flag(
-			"perflib.objects.add", "List of perflib objects to add to list").Uint32List()
+			"perflib.objects.add", "List of perflib object indices to add to list").Uint32List()
 		perfObjectsRemove = kingpin.Flag(
-			"perflib.objects.remove", "List of perflib objects to remove from list").Uint32List()
+			"perflib.objects.remove", "List of perflib object indices to remove from list").Uint32List()
+		perfObjectsNames = kingpin.Flag(
```

Comment:
README needs to be updated to reflect the new flags. Maybe add a short explanation that either indices or names can be used to the same effect.

Reply:
Done